docs/src/snap/howto/cis-hardening/etcd.md

```diff
@@ -0,0 +1,198 @@
+<!-- markdownlint-disable -->
+## Etcd Node Configuration
+### Etcd Node Configuration
+#### Control 2.1
+
+Description: `Ensure that the --cert-file and --key-file arguments are set as appropriate (Automated)`
+
+Audit:
+```
+/bin/ps -ef | /bin/grep etcd | /bin/grep -v grep
+```
+
+Remediation:
+```
+Follow the etcd service documentation and configure TLS encryption.
```

Comment:
I know text like this comes from the actual config. What scope do we have to change that upstream to make these work better

Reply:
We can change them. Let's schedule work for this next.